src/Layers/xrRender/HOM.cpp

```diff
@@ -124,7 +124,22 @@ void CHOM::Load()
 
     // Create AABB-tree
     m_pModel = xr_new<CDB::MODEL>();
-    m_pModel->build(CL.getV(), int(CL.getVS()), CL.getT(), int(CL.getTS()));
+    if (strstr(Core.Params, "-cdb_cache"))
+    {
+        strconcat(sizeof(fName), fName, "cdb_cache" DELIMITER, FS.get_path("$level$")->m_Add, "hom.bin");
```

Comment:
sizeof not needed. Will be substituted  automatically.
```suggestion
        strconcat(fName, "cdb_cache" DELIMITER, FS.get_path("$level$")->m_Add, "hom.bin");
```